### ISSUE TITLE
element qui ne s'affiche pas

### ISSUE THREAD

**Issue by cesio-althae** [hacf-fr/lovelace-meteofrance-weather-card#136]
Bonjour

j'ai installé le module sur mon Home Assistant tout neuf mais je ne vois pas les prévisions pour les jours suivants
![image](https://github.com/hacf-fr/lovelace-meteofrance-weather-card/assets/93390142/5f7689a0-f679-40bc-b4f7-a0e5ca955bc6)
voici ce que j'obtiens

Bien à vous

**Comment by DW-homeassistant** [comment]
Même problème depuis la MAJ Core 2024.4.x : les affichages heure par heure, ou jour par jour ne fonctionnent plus

**Comment by dx44** [comment]
Duplicate with #135

**Comment by diyanei** [comment]
C'est fonctionnel en version 1.9.2-beta-2 avec la 2024.4.x

my 2 cents.

**Comment by JohannCR** [comment]
@diyanei effectivement, merci pour l'info !
Par contre le sensor.weather n'a plus d'attribut forecast, une idée de comment récupérer les valeurs des prévisions journalières ?

**Comment by JohannCR** [comment]
Trouvé. Pour info, il faut créer un template sensor dans le fichier configuration.
[Exemple](https://community.home-assistant.io/t/new-weather-forecast-template/622642/109)

**Comment by Tos26** [comment]
> Trouvé. Pour info, il faut créer un template sensor dans le fichier configuration.
> [Exemple](https://community.home-assistant.io/t/new-weather-forecast-template/622642/109)

Bonjour,

Pouvez-vous être un peu plus précis, je ne sais pas exactement quoi faire dans le fichier de configuration avec la 1.9.2 beta-2 pour retrouver les prévisions journalières ?

Merci d'avance.

**Comment by Bastiencc** [comment]
Bonjour
Je veux bien plus d'info sur comment retrouver les détails jours et heure svp !

**Comment by Tos26** [comment]
@JohannCR : Serait-il possible d'avoir plus d'info détaillées pour refaire fonctionner cette intégration ? La version officielle est toujours en 1.9.0 et malgré la 1.9.2. beta, je n'arrive toujours pas à afficher les prévisions.

Merci.

**Comment by fcarlier** [comment]
Up !
Dans mon cas HACS ne veut pas mettre en version 1.9.2 beta, juste 1.9.0.
Toujours pas de prévision
Merci

**Comment by WarC0zes** [comment]
> Up ! Dans mon cas HACS ne veut pas mettre en version 1.9.2 beta, juste 1.9.0. Toujours pas de prévision Merci

Bonjour,
il faut utiliser la version beta 1.9.2-beta-2 pour avoir les précisions qui fonctionnent.
Mais il y a un problème d'installation de la beta sur HACS 2.0
https://github.com/hacf-fr/lovelace-meteofrance-weather-card/issues/140

**Comment by fcarlier** [comment]
"ne veut pas" -> "ne peut pas" mais @WarC0zes tu apportes le symptome avec #140 que je n'avais pas vu !!
Vient pas de moi alors, j'espère qu'une solution va être trouvé ;)
Merci...

**Comment by fcarlier** [comment]
Salut,
En passant par une installation manuelle, wget de la 1.9.2 beta en remplecement du répertoire existant ET en changeant le hacstag dans les ressources de ...190 en ...192, j'ai réussi à avoir la météo avec prévision.
Par contre dans le HACS, il me marque encore 1.9.0 et la page d'explication reste celle de 1.9.0, il doit y avoir une histoire de cache ou de duplication d'info qui ne se met pas à jour mais je ne connais pas encore bien la structure de HA.
Le plus simple étant une résolution proposée dans #140 ;)
Merci
A+

**Comment by Stef42300** [comment]
> Salut, En passant par une installation manuelle, wget de la 1.9.2 beta en remplecement du répertoire existant ET en changeant le hacstag dans les ressources de ...190 en ...192, j'ai réussi à avoir la météo avec prévision. Par contre dans le HACS, il me marque encore 1.9.0 et la page d'explication reste celle de 1.9.0, il doit y avoir une histoire de cache ou de duplication d'info qui ne se met pas à jour mais je ne connais pas encore bien la structure de HA. Le plus simple étant une résolution proposée dans #140 ;) Merci A+

Bonjour,

Tu n'as changé que le "0" par le "2" en fin d'URL?
Car moi cela ne marche pas.

Merci pour ton aide

**Comment by fcarlier** [comment]
Oui, j'ai changé en fin d'URL. J'ai aussi redémarrer HA. Après je ne sais pas si cela est la solution mais fonctionne.
J'ai aussi mis dans mon fichier configuration.yaml
`binary_sensor:
  - platform: meteoalarm
    country: "france"
    province: "Sarthe"`

**Comment by Stef42300** [comment]
Je viens de mettre les 2 nouveaux fichiers de la version 1.9.2. :
- meteofrance-weather-card
- meteofrance-weather-card-editor
J'ai mis à jour le lien URL dans les "Ressources" en modifiant le "190" par "192" à la fin mais je n'ai pas les prévisions.
A quoi sert le binary_sensor créé ?

Merci

**Comment by fcarlier** [comment]
Pour le binary_sensor, je ne sais plus ;) peut être pas de rapport.
Par contre tu as bien compléter ton configuration.yaml avec les nouveaux paramètres ?

**Comment by WarC0zes** [comment]
> Oui, j'ai changé en fin d'URL. J'ai aussi redémarrer HA. Après je ne sais pas si cela est la solution mais fonctionne. J'ai aussi mis dans mon fichier configuration.yaml `binary_sensor:
> 
> * platform: meteoalarm
>   country: "france"
>   province: "Sarthe"`
Salut,
ton binary_sensor, n'est pas utile pour faire fonctionner la carte MF.
C'est pour l'intégration meteoalarm ( https://www.home-assistant.io/integrations/meteoalarm/ ).

**Comment by fcarlier** [comment]
ah oui, c'est pour mon panel meteoalarm !! j'avais fait pleins de modif en même temps !! donc pas besoin dans le cas ici ;)

**Comment by fdecourt** [comment]
Pour ceux que ça branche d'avoir un truc qui semble fonctionner, j'ai fait un fork qui est opérationnel chez moi (avec les toutes dernières version de HACS) : 
mon dernier commentaire ici : https://github.com/hacf-fr/lovelace-meteofrance-weather-card/issues/148
